packages/mobile/src/app/ErrorBoundary.tsx

```diff
@@ -24,7 +24,7 @@ class ErrorBoundary extends React.Component<Props, State> {
 
   componentDidCatch(error: Error, info: any) {
     this.setState({ childError: error })
-    CeloAnalytics.track(DefaultEventNames.errorDisplayed, { error }, true)
+    ValoraAnalytics.track(AnalyticsEvents.error_displayed, { error: error.message })
```

Comment:
Could be useful to keep the device info for errors

Reply:
What we are doing now is exclusively capturing device info exclusively on app load. Device info rarely changes so the thought is we should only really log it once per session at a maximum. From there, we can tie in device info for any particular event by doing a join query on sessionId. This minimizes payload sizes, which should have positive performance + network implications.

Do you foresee any issues with this approach?